lib/assert.js

```diff
@@ -29,6 +29,7 @@ const {
   Map,
   NumberIsNaN,
   RegExpPrototypeTest,
+  Set,
```

Comment:
```suggestion
  SafeSet,
```

(We use a variant of `Set` that’s called `SafeSet` in our internal code whose methods cannot be overwritten by userland code, unlike the ones of `Set`)